### ISSUE TITLE
UKHPI - 1.1.1: Non-text Content

### ISSUE THREAD

**Issue by bogdanadrianmarc** [epimorphics/ukhpi#404]
See https://docs.google.com/spreadsheets/d/1YS3N5ps9pJGFOxBDi9C2DwA37WQPIVax/edit?gid=1215429860#gid=1215429860&range=5:5

**Comment by bogdanadrianmarc** [comment]
@tomguilbert Here's a screenshot of what's been done, notice the `alt="UKHPI Logo"` attribute:

![Image](https://github.com/user-attachments/assets/e677e33d-0b9d-4e6b-aab4-8535ca37d751)

Previously this was missing, and people using screen readers could not tell what was that image about, see https://www.w3schools.com/tags/att_img_alt.asp for more information.

There are two ways to test this, either using a screen reader or by visiting https://hmlr-dev-pres.epimorphics.net/app/ukhpi/ and inspecting the logo image using dev tools (Right Click on the logo -> Inspect, make sure the `<img>` element is selected, rather than the link)

**Comment by tomguilbert** [comment]
Thanks for the explanation Bogdan - looks good and I can see using the silktide accessibility checker that this no longer fails in dev.
